webserver/jws31-tomcat7-basic-s2i.json

```diff
@@ -4,10 +4,11 @@
     "metadata": {
         "annotations": {
             "iconClass": "icon-tomcat",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
             "description": "Application template for JWS applications built using S2I.",
-            "tags": "tomcat,tomcat7,java,jboss,xpaas",
+            "tags": "tomcat,tomcat7,java,jboss,hidden",
             "version": "1.4.6",
-            "openshift.io/display-name": "Red Hat JBoss Web Server 3.1 Tomcat 7 (no https)"
+            "openshift.io/display-name": "JBoss Web Server 3.1 Tomcat 7 (no https)"
```

Comment:
I don't know if we want to tackle this at the same time, but since we're changing it here, I believe we had a request to add "Apache" in front of Tomcat.

Reply:
updating